deepce.sh

```diff
@@ -1112,6 +1115,92 @@ exploitDockerSock() {
   # TODO: Tidy up command
 }
 
+exploitSysModule(){
+  caps=$(capsh --print)
```

Comment:
We need to add a check here, as `capsh` may not be installed. You can see this elsewhere in the code base:

```shell
if [ -x "$(command -v capsh)" ]; then
```

Reply:
Probably also worth testing the other requirements (make and insmod) are installed and showing an error if not telling them to install them?